cmd/site-replication.go

```diff
@@ -1031,6 +1031,35 @@ func (c *SiteReplicationSys) PeerAddPolicyHandler(ctx context.Context, policyNam
 	return nil
 }
 
+// PeerIAMUserChangeHandler - copies IAM user to local.
+func (c *SiteReplicationSys) PeerIAMUserChangeHandler(ctx context.Context, change *madmin.SRIAMUser) error {
+	var err error
+	if change.UserReq == nil {
+		err = globalIAMSys.DeleteUser(ctx, change.AccessKey, true)
+	} else {
+		err = globalIAMSys.CreateUser(ctx, change.AccessKey, *change.UserReq)
+	}
+	if err != nil {
+		wrapSRErr(err)
+	}
+	return nil
+}
```

Comment:
```suggestion
// PeerIAMUserChangeHandler - copies IAM user to local.
func (c *SiteReplicationSys) PeerIAMUserChangeHandler(ctx context.Context, change *madmin.SRIAMUser) error {
	var err error
	if change.UserReq == nil {
		err = globalIAMSys.DeleteUser(ctx, change.AccessKey, true)
	} else {
		err = globalIAMSys.CreateUser(ctx, change.AccessKey, *change.UserReq)
	}
	if err != nil {
		wrapSRErr(err)
	}
	return nil
}
```

To avoid errors can we do `change.DeleteReq != nil` instead of `UserReq == nil` ?  - remember the recent CVE.